### DISCUSSION TITLE
Dashboard: Add zoom-in shortcut for graphs

### DISCUSSION THREAD

**Discussion by xaviermerino** [grafana/grafana#38399] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
The ability to zoom in with a shortcut on graphs. 

**Why is this needed**:
There is a zoom-out shortcut (Ctrl+Z) but no zoom-in.

<img width="205" alt="image" src="https://user-images.githubusercontent.com/8650675/87578519-849b6700-c6a2-11ea-935c-c16f9b0d5d53.png">

Here's a list of current shortcuts (as of July 15, 2020, Grafana 7):

<img width="598" alt="image" src="https://user-images.githubusercontent.com/8650675/87578619-ac8aca80-c6a2-11ea-8531-3068e7037604.png">

**Comment by ivanahuckova** [comment]
Connected to https://github.com/grafana/grafana/issues/19194. Not sure if I would consider it duplicate.

**Comment by xaviermerino** [comment]
> Connected to #19194. Not sure if I would consider it duplicate.

The issue #19194 you are referring asks for a zoom-in button. This issue asks for a shortcut to that button. I guess they are related. 

:)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by vxe** [comment]
hi what is the status of this issue, having a zoom out without a corresponding zoom in keyboard shortcut seems like an important missing feature

**Comment by readonly24** [comment]
Any news?

**Comment by leeoniya** [comment]
no news.

it's not actually clear how valuable this feature would be. zooming in (assuming it's to the middle) will always chop off some data at the edges. it will only work perfectly if the data you're interested in is mostly in the middle, which does not feel like an 80% case, or even 50% case.

zooming out on the other hand will never result in moving the existing data out of view, but instead always provides more context.

i'm curious what types of workflows can frequently make use of a zoom-into-middle, and by what factor?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
